requirements/runtime.txt

```diff
@@ -1,2 +1,3 @@
 celery
 Django>=1.11.17
+ephem
```

Comment:
Is this just for the Docker app?

Reply:
`ephem` is needed to use solar scheduling. It's in the test requirements file so my guess is they added it for testing, everything looked ok, and then they forgot to add it to the runtime file.

If you try and use `celery.schedules.solar` it throws an ImportError.